internal/provider/space_resource.go

```diff
@@ -163,8 +163,8 @@ func (r *SpaceResource) Schema(ctx context.Context, req resource.SchemaRequest,
 				Required:            true,
 				Validators: []validator.String{
 					stringvalidator.RegexMatches(
-						regexp.MustCompile(`^[a-z]([\d-_]|[a-z]){6,35}[a-z\d]$`),
-						"must contain 6 to 35 digits, dashes, underscores, or lowercase letters, and ending with either a lowercase letter or a digit",
+						regexp.MustCompile(`^[a-z\d]([\d-_]|[a-z]){4,48}[a-z\d]$`),
+						"must contain 4 to 50 digits, dashes, underscores, or lowercase letters, and ending with either a lowercase letter or a digit",
```

Comment:
its 6 to 50

Reply:
oh, I'll have to fix the org message too, thank!